QATCH/VisQAI/src/view/evaluation_ui.py

```
@@ -678,35 +681,39 @@ def select_model(self) -> None:
+    def model_selected(self, path: Optional[str] = None) -> None:
+        """Handle the event when a model is selected from the file dialog."""
+        try:
+            file_path = path
+            self.model_path = file_path
+            self.predictor = Predictor(file_path, mc_samples=300)
+            display_name = file_path.split(
+                '\\')[-1].split('/')[-1].split('.')[0]
+            self.select_model_label.setText(display_name)
+            Log.i(TAG, f"Model loaded: {file_path}")
+            self.check_ready_to_evaluate()
+        except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)